jenkins/spark-tests.sh

```diff
@@ -276,6 +279,14 @@ fi
 
 # cudf_udf_test
 if [[ "$TEST_MODE" == "ALL" || "$TEST_MODE" == "CUDF_UDF_ONLY" ]]; then
+  if [[ "$IS_SPARK_320_OR_LATER" -eq "1" ]]; then
```

Comment:
Does this change need to be specific to Spark 3.2?  If pulling in the conda version of pandas works across all Spark versions, I'd rather reduce the number of special setups we have to maintain.

Reply:
I think it's because there's no pandas package in the python folder of the previous version of spark 3.2.
But yes, I think it should be no harm to do in all the spark.

Reply:
There is another spark `pandas` folder in spark-3.2.0 or later versions, so we use conda `pandas` to overwrite spark `pandas`.

No harm to do in all the spark, updated